samples/client/others/rust/hyper/emptyObject/src/models/empty_object.rs

```diff
@@ -8,6 +8,7 @@
  * Generated by: https://openapi-generator.tech
  */
 
+use crate::models;
```

Comment:
A lot of client files contain only this change - it's now aligned with server-side generators (via `AbstractRustCodegen`).